src/stylesheet.css

```diff
@@ -12,40 +12,83 @@ html {
 
 body {
 	padding: 1rem;
-	font-size: 16px;
+	font-size: 1rem;
 	line-height: 1.5;
-	--link-color: #C1D8FC
+	--link-color: #fff
```

Comment:
This should probably be `inherit` if the expectation is that it takes the colour of its parent.